test/integration/replication.bats

```diff
@@ -185,6 +185,7 @@ function containerRunning() {
 	# Bring up one manager, make sure it becomes primary.
 	swarm_manage --replication --replication-ttl "4s" --advertise 127.0.0.1:$SWARM_BASE_PORT "$NODE_1_URL"
 	run docker -H ${SWARM_HOSTS[0]} info
+	echo "$output"
```

Comment:
The issue is not resolved so I want to keep it here. Just to be sure it's the same error.